chainercv/links/model/deeplab/deeplab_v3_plus.py

```diff
@@ -0,0 +1,278 @@
+from __future__ import division
+
+import chainer
+import chainer.functions as F
+import chainer.links as L
+
+from chainercv.links.connection import Conv2DBNActiv
+from chainercv.links.connection import SeparableConv2DBNActiv
+from chainercv.links.model.deeplab.aspp import SeparableASPP
+from chainercv.links.model.deeplab.xception import Xception65
+from chainercv.transforms import resize
+from chainercv import utils
+
+import numpy as np
+
+
+class Decoder(chainer.Chain):
+
+    """Decoder for DeepLab V3+.
+
+    Args:
+        in_channels (int): Number of channels of input arrays.
+        out_channels (int): Number of channels of output arrays.
+        proj_channels (int): Number of channels of output of
+            first 1x1 convolution.
+        depth_channels (int): Number of channels of output of
+            convolution after concatenation.
+        bn_kwargs (dict): Keyword arguments passed to initialize
+            :class:`chainer.links.BatchNormalization`.
+
+    """
+
+    def __init__(self, in_channels, out_channels, proj_channels=48,
+                 depth_channels=256, bn_kwargs={}):
```

Comment:
I am not sure how the default value of `proj_channels` and `depth_channels` are set.
Is is possible to not set them?